tests/models/mask2former/test_image_processing_mask2former.py

```diff
@@ -242,6 +269,15 @@ def common(is_instance_map=False, segmentation_type=None):
         common(is_instance_map=True)
         common(is_instance_map=False, segmentation_type="pil")
         common(is_instance_map=True, segmentation_type="pil")
+        common(num_channels=1, numpify=True)
+        common(num_channels=2, numpify=True, input_data_format=ChannelDimension.LAST)
+        common(num_channels=5, numpify=True, input_data_format=ChannelDimension.LAST, do_resize=False)
```

Comment:
Can you please make a call with explicit `input_data_format=ChannelDimension.FIRST` for better readability of the test?